### PR TITLE
fixes nanites

### PR DESCRIPTION
**Pull request by zeskorion**
## About The Pull Request
ok so basically last minute code suggestion from archanial broke everything, this reverts that so nanites work again lol
## Why It's Good For The Game
if its stupid and it works it aint stupid

## Changelog
:cl:
fix: fixes nanites
/:cl:

### CODE REVIEW

**Comment by OceanFish1** [comment]
test your PRs kids test your PRs kids test your PRs kids test your PRs kids test

**Comment by zeskorion** [comment]
> test your PRs kids test your PRs kids test your PRs kids test your PRs kids test

it was tested tho, then a last minute change happend cuz of SOMEONE being a fuckcupcake

**Comment by Kapu1178** [comment]
(This PR reverts nanites to the **version I tested** prior to a code suggestion pre-merge.

**Comment by utilbird** [comment]
@zeskorion Actually, shouldn't it be `var/static/list` not `var/list/static`

**Comment by zeskorion** [comment]
> @zeskorion Actually, shouldn't it be `var/static/list` not `var/list/static`

This is taken from the format used for other typed nanites. idk.

im afraid of making any more changes that break shit tho

File: code/modules/research/nanites/nanite_programs.dm [hunk 1]
```diff
@@ -57,10 +57,10 @@
 	//Logic
 	//a list of logic types a nanite program's rules follow
 	var/list/static/logic = list(
```
Comment (ike709) [inline code/modules/research/nanites/nanite_programs.dm]:
```suggestion
	var/static/list/logic = list(
```

**Review by ike709 — APPROVED** [review]

**Pull request merged**